hal/ia32/_interrupts.S

```diff
@@ -81,6 +89,16 @@ interrupts_pushContext:
 .align 4, 0x90
 interrupts_popContext:
 	popl %esp
+	_INTERRUPTS_MULTILOCKCLEAR %eax
```

Comment:
interrupts_popContext might be called without mulilock (e.g. line 153), in such case we musn't clear it